torchvision/transforms/transforms.py

```diff
@@ -1172,7 +1172,13 @@ class ColorJitter(torch.nn.Module):
             or use an interpolation that generates negative values before using this function.
     """
 
-    def __init__(self, brightness=0, contrast=0, saturation=0, hue=0):
+    def __init__(
+        self,
+        brightness: "float | tuple[float, float]" = 0,
```

Comment:
As far as I know, using the `|` operator for `Union` is only supported starting Python 3.10. See [PEP604](https://peps.python.org/pep-0604/#extend-isinstance-and-issubclass-to-accept-union) for details. Is this the reason you are using string annotations here?

Reply:
Kind of. I think using `tuple` instead of `Tuple` from `typings` does not work on python 3.8 either (which I'm still using).

Reply:
Yeah, our minimum version is 3.7 so we'll have to abide by that.